tutorials/fundamentals/object-destructuring-rename.md

```diff
@@ -0,0 +1,11 @@
+When destructuring an object, you can rename the variables like so:
+
+```javascript
+const obj = { first_name: 'John', last_name: 'Smith' };
+
+const { first_name: firstName, last_name: lastName } = obj;
+firstName; // 'John'
+lastName; // 'Smith'
+```
+
+This is perfect for making sure your variable names are in line with the standards used at your workplace or making a vague variable name provided from an http request more clear.
```

Comment:
"This is perfect for cases where you want to rename variables from a REST API whose naming conventions don't line up with yours. For example, switching variables from `snake_case` to `camelCase` or vice versa."